includes/Core/Modules/Modules.php

```diff
@@ -566,10 +566,13 @@ protected function get_registry() {
 	 * @return Module_Registry
 	 */
 	protected function setup_registry() {
-		$registry = new Module_Registry();
+		$registry          = new Module_Registry();
+		$available_modules = (array) apply_filters( 'googlesitekit_available_modules', array_keys( $this->core_modules ) );
```

Comment:
We need to add a documentation block here (all actions/filters should have one). As mentioned in parts in https://github.com/google/site-kit-wp/issues/3993#issuecomment-992629451, let's also include in that documentation that force-active modules cannot be modified by this (see also comment below).